test/functional/fixtures/regression/gh-973/test.js

```diff
@@ -0,0 +1,26 @@
+describe('[Regression](GH-973) ', function () {
+    describe('Should leave extra distance between target and browser window while performing an action', function () {
```

Comment:
describe('Should leave an extra distance between the target and browser window while performing an action',